### PR TITLE
fix(format): should format success on null prototyped object

### LINKED ISSUE #166: format null prototyped object throw unexpected error
##  Step to reproduce: 

1. write code as below in `index.mjs`: 
```
// index.mjs
import { EdgeRuntime } from 'edge-runtime'
const main = async () => {
  try {
    const runtime = new EdgeRuntime();
    runtime.evaluate(`
      const a  = Object.create({})
      console.log('a --> ', a);
      const b  = Object.create(null)
      console.log('b --> ', b);
    `);
  } catch(err) {
    console.log(`evaluate error: `, err);
  }
}
main();
``` 
2. executing `node ./index.mjs`, you can see the error in terminal: 

```
evaluate error:  evalmachine.<anonymous>:124
    var getConstructorName = /* @__PURE__ */ __name((object) => object.constructor.name, "getConstructorName");
                                                                                   
TypeError: Cannot read properties of undefined (reading 'name')
    at getConstructorName (evalmachine.<anonymous>:124:84)
    at formatRaw (evalmachine.<anonymous>:229:29)
    at formatValue (evalmachine.<anonymous>:224:16)
    at inspect (evalmachine.<anonymous>:346:16)
    at format (evalmachine.<anonymous>:196:26)
    at Object.log (evalmachine.<anonymous>:571:55)
    at evalmachine.<anonymous>:5:15
```
## Expected
`console.log` should work successfully on objects with null prototype as it does in the browser.

## Environments
* node: v16.15.1
* edge-runtime: 1.1.0-beta.31

### PR DESCRIPTION
**Pull request by await-ovo**
To Fix #166

### CODE REVIEW

File: packages/format/tests/index.test.ts [hunk 1]
```diff
@@ -7,6 +7,16 @@ it('first argument', () => {
   expect(format('')).toBe('')
   expect(format([])).toBe('[]')
   expect(format({})).toBe('{}')
+  expect(format(Object.create(null))).toBe('{}')
```
Comment (Kikobeats) [inline packages/format/tests/index.test.ts:10]: In Node.js, this is printed as `[Object: null prototype] {}` 🤔

**Comment by Kikobeats** [comment]
Hello @await-ovo, thanks for taking care about this 🙂

I added a commit to differentiate when an object has null prototype, since it's formatted in a different way. 

How it looks to you?

File: packages/format/tests/index.test.ts (hunk 1 above)
Comment (await-ovo) [reply packages/format/tests/index.test.ts]: Thank you very much, I didn't notice this, I just tested the case on the browser, but I just tried it in Deno and it looks like this:

image

Ummm, I don't know which one is better for us.

**Comment by await-ovo** [comment]
> Hello @await-ovo, thanks for taking care about this 🙂
> 
> I added a commit to differentiate when an object has null prototype, since it's formatted in a different way.
> 
> How it looks to you?

Thanks, It looks good to me~

**Review by feugy — APPROVED** [review]
Hello @await-ovo, and thanks for both reporting and fixing this issue.

So far, we took inspiration from Node's formatting behavior, so let's stick with it. Besides, it's a bit more detailed (telling use when the created object is null).

**Pull request merged**